guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-manually.adoc

```diff
@@ -0,0 +1,63 @@
+[id="deploying-the-ca-certificate-on-a-host-manually"]
+= Deploying the CA certificate on a host manually
+
+You can deploy the CA certificate on the host manually by rendering a public provisioning template, which will provide the certificate to your host.
+
+.Prerequisites
+* You have root access on your host.
+* The CA certificate has been changed on {ProjectServer}.
+For more information, see {AdministeringDocURL}planning-for-ca-certificate-renewal[Planning for CA certificate renewal] in _{AdministeringDocTitle}_.
+
+.Procedure
+. Log in to you host by using SSH.
+. Download the certificate on your host:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# curl -o "{project-context}_ca_cert.crt" https://_{foreman-example-com}_/unattended/public/foreman_raw_ca
+----
++
+include::snip_replace-fqdn-projectserver.adoc[]
++
+You can use HTTP when the CA certificate is expired.
+However, {Team} recommends preparing the certificate renewal in advance.
+. Copy the certificate to the truststore:
+ifdef::client-content-dnf[]
+ifndef::satellite[]
+* On {EL}:
+endif::[]
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# cp {project-context}_ca_cert.crt /etc/pki/ca-trust/source/anchors
+----
+endif::[]
+ifdef::client-content-apt[]
+* On {DL}:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# cp {project-context}_ca_cert.crt /usr/local/share/ca-certificates/
+----
+endif::[]
+. Update the truststore:
+ifdef::client-content-dnf[]
+ifndef::satellite[]
+* On {EL}:
+endif::[]
```

Comment:
I'd combine those steps into 2 larger `ifdef` blocks. So rather than
```asciidoc
ifdef::client-content-dnf[]
. copy step
endif::[]
ifdef::client-content-apt[]
. copy step
endif::[]
ifdef::client-content-dnf[]
. update step
endif::[]
ifdef::client-content-apt[]
. update step
endif::[]
```

I'd prefer

```asciidoc
ifdef::client-content-dnf[]
. copy step
. update step
endif::[]
ifdef::client-content-apt[]
. copy step
. update step
endif::[]
```

Reply:
I'd prefer to keep the order of the steps.